istioctl/pkg/multicluster/generate.go

```diff
@@ -28,17 +28,17 @@ import (
 	ktypes "k8s.io/apimachinery/pkg/types"
 
 	"istio.io/api/mesh/v1alpha1"
-	operatorV1alpha1 "istio.io/operator/pkg/apis/istio/v1alpha1"
-	operatorV1alpha2 "istio.io/operator/pkg/apis/istio/v1alpha2"
-	"istio.io/operator/pkg/util"
-	"istio.io/operator/pkg/validate"
+	iop "istio.io/api/operator/v1alpha1"
+	operatorV1alpha1 "istio.io/istio/operator/pkg/apis/istio/v1alpha1"
+	"istio.io/istio/operator/pkg/util"
+	"istio.io/istio/operator/pkg/validate"
 
 	"istio.io/istio/pilot/pkg/serviceregistry"
 	"istio.io/istio/pkg/util/protomarshal"
 )
 
 // defaults the user can override
-func defaultControlPlane() (*operatorV1alpha2.IstioControlPlane, error) {
+func defaultControlPlane() (*operatorV1alpha1.IstioOperator, error) {
```

Comment:
Why are we going back a version here?

Reply:
I don't know why it went back, but this is the newer version. We unfortunately hadn't fully synced up the operator dep when we did the repo merge, so we have some extra changes like this

Reply:
operatorV1alpha2 is a private API, it was never officially released in istio/api.